benchmark/benchmark.cpp

```diff
@@ -182,17 +171,13 @@ UMF_BENCHMARK_TEMPLATE_DEFINE(multiple_malloc_free_benchmark, jemalloc_pool_fix,
                               fixed_alloc_size,
                               pool_allocator<jemalloc_pool<os_provider>>);
 UMF_BENCHMARK_REGISTER_F(multiple_malloc_free_benchmark, jemalloc_pool_fix)
-    ->Args({10000, 4096})
-    ->Threads(4)
-    ->Threads(1);
+    ->Apply(&default_multiple_alloc_fix_size);
 
 UMF_BENCHMARK_TEMPLATE_DEFINE(multiple_malloc_free_benchmark,
                               jemalloc_pool_uniform, uniform_alloc_size,
                               pool_allocator<jemalloc_pool<os_provider>>);
 UMF_BENCHMARK_REGISTER_F(multiple_malloc_free_benchmark, jemalloc_pool_uniform)
-    ->Args({1000, 8, 64 * 1024, 8})
-    ->Threads(4)
-    ->Threads(1);
+    ->Apply(&default_multiple_alloc_uniform_size);
```

Comment:
sam here, no extra `0` before

Reply:
and still no zero - it is default_multiple_alloc_uniform_size function

Reply:
perfect, thx for double checking